tonic-health/src/lib.rs

```diff
@@ -28,6 +28,10 @@ pub mod proto {
     #![allow(unreachable_pub)]
     #![allow(missing_docs)]
     tonic::include_proto!("grpc.health.v1");
+
+    #[allow(dead_code)]
```

Comment:
if this is public why do we need dead_code?

Reply:
You are right, I went ahead and removed that.